src/actions/deviceActions.ts

```diff
@@ -449,6 +454,7 @@ export function triggerSingleSet() {
         logger.info(`Waiting for single trigger at ${triggerLevel} \u00B5A`);
 
         await device!.ppkTriggerSingleSet();
+        startPreventSleep();
```

Comment:
Is in this case `stopPreventSleep()` also called eventually? Or do we also need to add it somewhere, e.g. in https://github.com/NordicSemiconductor/pc-nrfconnect-ppk/blob/a424151e9489fc9a34cf3cf23df547eab21e3dc5/src/actions/triggerActions.ts#L81-L85?